embulk-cli/src/main/java/org/embulk/cli/EmbulkMigrate.java

```diff
@@ -25,6 +25,8 @@
 
 import org.apache.maven.artifact.versioning.ComparableVersion;
 
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
```

Comment:
Please import just `java.nio.file.StandardCopyOption`, and refer like `StandardCopyOption.REPLACE_EXISTING`.